src/pocketmine/MemoryManager.php

```diff
@@ -151,7 +151,7 @@ public function isLowMemory() : bool{
 	 * @return bool
 	 */
 	public function canUseChunkCache() : bool{
-		return !($this->lowMemory and $this->chunkTrigger);
+		return !$this->lowMemory or !$this->chunkCache;
```

Comment:
the field name is very confusing, I'm open to a better name. It actually indicates whether the chunk cache should be disabled on low memory.

Just to verify your logic:
the old condition: NOT low memory, AND clear cache on low memory. (bad, nonsense?)
the new condition: NOT low memory, OR NOT disable chunk cache on low memory. (good)

You can see how this would be confusing to future viewers.